extensions/wikia/TrackingOptIn/TrackingOptIn.class.php

```diff
@@ -12,6 +12,7 @@ public static function onOasisSkinAssetGroupsBlocking( &$jsAssets ) {
 
 	public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgEnableTrackingOptInModal';
+		// TODO: Remove variable once @wikia/tracking-opt-in#1.0.36 is released
```

Comment:
Do we have ticket for the release or it's all within ADEN-7432 but a different step?